src/data/nat/choose.lean

```diff
@@ -99,4 +99,8 @@ theorem add_pow [comm_semiring α] (x y : α) (n : ℕ) :
   (x + y) ^ n = ∑ m in range (n + 1), x ^ m * y ^ (n - m) * choose n m :=
 (commute.all x y).add_pow n
 
+theorem sum_pascal_row (n : ℕ) :
+  finset.sum (finset.range n.succ) (choose n) = 2 ^ n :=
```

Comment:
```suggestion
/-- Summing a row of Pascal's triangle -/
theorem sum_range_choose (n : ℕ) :
  ∑ m in range (n + 1), choose n m = 2 ^ n :=
```
I think this name is more along the lines of [our naming conventions](https://leanprover-community.github.io/contribute/naming.html).